README.md

```diff
@@ -1,3 +1,5 @@
+![IBM Turorials Logo](./images/tutorialslogo.png)
```

Comment:
@aspleenic can you just fix the typo. Otherwise its great :)

Reply:
I thought this could be our clever rename :)